### ISSUE TITLE
[ProofPDF.py] Help needs update

### ISSUE THREAD

**Issue by Arno-Enslin** [adobe-type-tools/afdko#123]
The "showfont" command file is not part of the AFDKO anymore.

By the way: I wonder, whether it is senseful to write about the command files in the help of ProofPDF.py. The command files could have their own help instead.

```
"charplot",  "digiplot", "fontplot", "hintplot", and "showfont" are all
command files that call the ProofPDF script with different options.
ProofPDF takes as options a list of fonts, and an optional list of
glyphs, and prints a PDF file for the specified font. showing the glyphs
as specified by options.

The five main options, "-charplot", "-digiplot", "-fontplot", and
"-hintplot"", each set a bunch of lower level parameters in order to
produce a particular page layout. All these low-level parameters can be
set by command-line options. This means that you can edit one of the
command files to add your own parameter values to make your own
customized proofing command. The "showfont" command file is an example
of adding additional low-level parameter specifications to the -fontplot
option.
```

**Comment by frankrolf** [comment]
It would be helpful if pull requests were opened for suggestions like this one. 
Please suggest how the help content should be re-worded, and it is likely your PR will be accepted.

**Comment by Arno-Enslin** [comment]
I think I would do it in this way:

First I would rename:

digiplot.cmd to ProofPDF_ digiplot.cmd
fontplot.cmd to ProofPDF_ fontplot.cmd
fontsetplot.cmd to ProofPDF_ fontsetplot.cmd
hintplot.cmd to ProofPDF_ hintplot.cmd
waterfallplot.cmd to ProofPDF_ waterfallplot.cmd

Then I would make these files able to display the help of ProofPDF on double click / command without arguments. Or alternatively: A short explanation about the function of the command files, which is contained in them and which is displayed on double click. And if you start the command files with the arguments -digiplot, -fontplot, -fontsetplot, -hintplot or -waterfallplot or -h, but without any other arguments, the help of ProofPDF is displayed.

On the other side I don't make use of these command files, but write my own ones anyway. **My point is mainly, that the help of ProofPDF.py actually mentions showfont.cmd, although this file does not exist anymore in the AFDKO.**

**Comment by frankrolf** [comment]
🤔 
> display the help of ProofPDF
Doesn’t that kind of already happen? Any of the commands with the `-h` option prints that same help file. Why renaming the files? That would make them harder to use.

> displayed on double click
Those files are not intended for double-clicking. Fair enough, Windows will execute them with cmd, but that is not their main purpose. Using them on the command line is the preferred scenario.

> the help of ProofPDF.py actually mentions showfont.cmd
Please fix this by submitting a Pull Request for showfont.cmd.

**Comment by Arno-Enslin** [comment]
> Why renaming the files? That would make them harder to use.

That is true, if you manually write the commands into the command line.

> Please fix this by submitting a Pull Request for ProofPDF.py

I will try. But wouldn't it make more sense, if the developers of the AFDKO would think about my proposal to source the info about the command files out at first?

I mean something like this. As you can see, the command file contains an own help info. And after the echo of that info the help of ProofPDF.py is "called". In this case the complete info about the command files could be removed from ProofPDF.py.

```
@echo off
rem v 2.6 Feb 3 2012

rem See comments in setFDKPaths.cmd

call "%~dp0setFDKPaths.cmd"

if [%1]==[] goto :help
if [%1]==[-h] goto :help

%AFDKO_Python% %AFDKO_SCRIPTS%\ProofPDF.py -charplot %*
goto :end

:help
echo "charplot", "digiplot", "fontplot", "hintplot", and "showfont" are all
echo command files that call the ProofPDF script with different options.
echo ProofPDF takes as options a list of fonts, and an optional list of
echo glyphs, and prints a PDF file for the specified font. showing the
echo glyphs as specified by options.
%AFDKO_Python% %AFDKO_SCRIPTS%\ProofPDF.py -h

:end
pause
exit
```

**Comment by readroberts** [comment]
The reference to 'showfont' is just a typo - it was renamed to fontplot2 in 2009, and fontplot2 does exist. I will fix this.  I also think that it would be better to have command specific help.  I'd like to see this with -u, and the full general set of options only displayed if with -h. I'd rather put the command specific help in ProofPDF.py, though, so that changes can be made in only one place. If we put the help in the command  files, then any change to the text needs to be made in three different command files - Windows Linux and OSX.

**Comment by Arno-Enslin** [comment]
> The reference to 'showfont' is just a typo - it was renamed to fontplot2 in 2009, and fontplot2 does exist.

Only the argument "-fontplot2", but no command file "fontplot2.cmd".

Good to know. This would mean, that

`-fontplot --drawGlyph_CenterMarksWithBox 1 --drawGlyph_EMBox 0 --drawGlyph_XAdvance 0 --drawMeta_SideBearings 0` (Showfont arguments)

and

-fontplot2

have identical results, wouldn't it? I will compare that.

**Comment by miguelsousa** [comment]
Fixed by 34925024191ccda77ac60491ff1376e6abeaf70f
